src/App.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+
+// ? Components
+import Navbar from './components/Navbar';
+import Books from './components/Books';
+import Categories from './components/Categories';
+import Form from './components/Form';
```

Comment:
- [ ] Please as it is a requirement for this project, your `Form component` is supposed to rendered in your  `Books component`. See screenshot of the requirements below for clarifications and kindly fix that.

![bookComponent](https://user-images.githubusercontent.com/62849872/155263923-73a769c3-0963-4bb5-ae9d-5c43d5f0cfab.png)

Reply:
This issue is well resolved! ✔️